Picnic.Net.Dashboard/GoogleService/Services/GoogleCalendar.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Calendar.v3.Data;
+using Google.Apis.Services;
+using Google.Apis.Util.Store;
+using GoogleService.Boundary;
+using GoogleService.Models;
+
+
+namespace GoogleService.Services
+{
+    public class GoogleCalendar : GoogleService, IGoogleService
+    {
+        
+        private readonly string ApplicationName = "Dashboard";
+        private const string API_KEY = "AIzaSyAFtGoV2zO6NgXF9XLFu0kTyGr6L_2ND8M";
```

Comment:
Consider moving API Key to some config file - it could be replaced in the future quickly without building the whole solution